front/src/features/filter/TimeDropdown.tsx

```diff
@@ -1,12 +1,14 @@
+import { Dropdown } from "@/components/molecules/Dropdown";
 import { useNavigation } from "@/hooks/useNavigation";
-import { Dropdown } from "../atoms/Dropdown";
+import { TimeOptions } from "@/types/filter";
 
 export const TimeDropdown = () => {
     const { searchParams, updateSearchParam } = useNavigation();
     const timeOptions = ["Last hour", "Last day", "Last week"];
-    const selectedTime = searchParams.get("dateTimeFilter") || "Last hour";
+    const selectedTime =
+        (searchParams.get("dateTimeFilter") as TimeOptions) || "Last hour";
```

Comment:
What if the type isn't matching `TimeOptions` and you've cast it incorrectly here?

Are there any side-effects? Any error handling needed?